app/e2e/packetPage.spec.ts

```diff
@@ -0,0 +1,118 @@
+          expect((await link.getAttribute("href")).split("/").at(-1)).toBe(linkText);
+        }
+      }
+    });
+
+    test("can see Reports section", async ({ page }) => {
+      const content = await getContentLocator(page);
+      const reportsDiv = await getPacketPageAccordionSection(content, "Reports");
+      // Reports should contain either "None" or at least one iframe
+      const noneDiv = await reportsDiv.locator(".italic");
+      if ((await noneDiv.count()) > 0) {
+        await expect(noneDiv).toHaveText("None");
+      } else {
+        const reportFrames = await reportsDiv.locator("iframe");
+        await expect(await reportFrames).toHaveCount(1);
+      }
+    });
+  });
+
+  test.describe("Metadata", () => {
+    test("can see Timings section", async ({ page }) => {
+      const content = await getContentLocator(page);
+      await selectPacketPageTab(content, "Metadata");
+      const timingsDiv = await getPacketPageAccordionSection(content, "Timings");
+      const timingItems = await timingsDiv.getByRole("listitem");
+      await expect(await timingItems).toHaveCount(2);
+      await expect(await timingItems.first()).toHaveText(/Started/);
+      await expect(await timingItems.last()).toHaveText(/Elapsed\d+/);
+    });
+
+    test("can see Git section", async ({ page }) => {
+      const content = await getContentLocator(page);
+      await selectPacketPageTab(content, "Metadata");
+      const gitDiv = await getPacketPageAccordionSection(content, "Git");
+      // get top level list items
+      const listItems = await gitDiv.locator("ul.space-y-1 > li").all();
+      await expect(await listItems.length).toBe(3);
+      await expect(listItems[0]).toHaveText(/^Branch/);
+      await expect(listItems[1]).toHaveText(/^Commit[\da-f]{40}$/);
+      await expect(await listItems[2].locator("span")).toHaveText("Remotes");
+      const remoteItems = await listItems[2].getByRole("listitem");
+      await expect(await remoteItems.count()).toBeGreaterThan(0);
+    });
+  });
+
+  test("can see Platform section", async ({ page }) => {
+    const content = await getContentLocator(page);
+    await selectPacketPageTab(content, "Metadata");
+    const platformDiv = await getPacketPageAccordionSection(content, "Platform", true);
+    // get top level list items
+    const listItems = await platformDiv.locator("ul.space-y-1 > li").all();
+    await expect(listItems.length).toBe(3);
+    await expect(listItems[0]).toHaveText(/^OS/);
+    await expect(listItems[1]).toHaveText(/^System/);
+    await expect(listItems[2]).toHaveText(/^Language/);
+  });
+
+  test("can see Packages section", async ({ page }) => {
+    const content = await getContentLocator(page);
+    await selectPacketPageTab(content, "Metadata");
+    const packagesDiv = await getPacketPageAccordionSection(content, "Packages", true);
+    // Might be empty but expect the list to exist!
+    await expect(await packagesDiv.locator("ul.space-y-1")).toBeVisible();
+  });
```

Comment:
NB the 'session' metadata, used here, is required only for orderly packets, not outpack packets (maybe just add this as a comment in case we never in fact have any non-orderly packet)